src/vector/index.js

```diff
@@ -30,7 +30,6 @@ require('babel-polyfill');
 
 // CSS requires: just putting them here for now as CSS is going to be
```

Comment:
Not sure that's an excuse for making it worse: for a start I think it is no longer the case that CSS is going to be refactored soon, since I think this is the refactoring alluded to.

Perhaps @dbkr can clarify, but I think you could getting rid of this comment and replace it with:

"Require common CSS for our dependencies here; this will make webpack process it into bundle.css. Our own CSS (which is themed) is imported via separate webpack entry points in webpack.config.js"